src/hastings_index.erl

```diff
@@ -339,6 +339,10 @@ handle_info({'EXIT', Pid, Reason}, #st{index=#h_idx{pid={_, Pid}}} = St) ->
     couch_log:info(Fmt, [index_name(St#st.index), Reason]),
     [gen_server:reply(P, {error, Reason}) || {P, _} <- St#st.waiting_list],
     {stop, normal, St};
+handle_info({'EXIT', Pid, Reason}, St) ->
+    % probably hastings_index_manager.
+    couch_log:notice("Unknown pid ~p closed with reason ~p", [Pid, Reason]),
```

Comment:
I noticed that in all other cases where we handle `EXIT` we do `gen_server:reply`. Why not in this case?

Reply:
Following the same approach in dreyfus https://github.com/cloudant-labs/dreyfus/blob/master/src/dreyfus_index.erl#L228-L231. My understanding is that it is not necessary to do `gen_server:reply` to pid in `waiting_list`.